integration/client_sdk/go/utils.go

```diff
@@ -97,3 +115,158 @@ func setupNetwork(channel string) (*gateway.Network, error) {
 
 	return network, nil
 }
+
+func setup(ccID, ccPath string) error {
+	nwDetails := &testutils.NetworkDetails{
+		ChannelID: "mychannel",
+		Peers:     []string{"jdoe_test.sampleorg.example.com"},
+		Orderers:  []string{"orderer.example.com"},
+	}
+
+	mrenclave, err := fpc.ReadMrenclave(ccPath)
+	if err != nil {
+		return err
+	}
+
+	ccDetails := &testutils.ChaincodeDetails{
+		Id:           ccID,
+		Path:         ccPath,
+		Version:      mrenclave,
+		Lang:         fpc.ChaincodeLang,
+		Policy:       policydsl.SignedByAnyMember([]string{"SampleOrg"}),
+		Seq:          int64(1),
+		Vscc:         "vscc",
+		Escc:         "escc",
+		InitRequired: false,
+	}
+
+	// get sdk instance
+	sdk, err := fabsdk.New(config.FromFile(filepath.Clean(ccpPath)))
+	if err != nil {
+		return fmt.Errorf("failed to create sdk: %v", err)
+	}
+	defer sdk.Close()
+
+	// new client
+	orgAdmin := "Admin"
+	orgName := "SampleOrg"
+
+	adminContext := sdk.Context(fabsdk.WithUser(orgAdmin), fabsdk.WithOrg(orgName))
+
+	client, err := fpcmgmt.New(adminContext)
+	if err != nil {
+		return fmt.Errorf("failed to create context: %v", err)
+	}
+
+	// install fpc chaincode
+	err = installChaincode(client, ccDetails, nwDetails)
+	if err != nil {
+		return fmt.Errorf("error during installing chaincode: %v", err)
+	}
+
+	return nil
+}
+
+func installChaincode(client *fpcmgmt.Client, cc *testutils.ChaincodeDetails, nw *testutils.NetworkDetails) error {
+	// package
+	desc := &fpc.Descriptor{
+		Path:  cc.Path,
+		Type:  cc.Lang,
+		Label: cc.Id,
+	}
+	ccPkg, err := fpc.NewCCPackage(desc)
+	if err != nil {
+		return fmt.Errorf("failed to create new chaincode package: %v", err)
+	}
+	logger.Infof("%s successfully packaged", cc.Id)
```

Comment:
This is how packaging is supposed to be used.

Reply:
Main suggestion i would have regarding naming:
- instead `Descriptor` call it `...NewCCPackageRequest`
- prefix request type and operation with `Lifecycle`
This would make it even more uniform with the other operations.

The other thought i would have is whether the sgx-mode we might expose also in the request (maybe still with the env-var based derivation if it is not explicitly set in request)?

On a related note, don't forget to add godoc comments!

Reply:
Regarding naming, I was following the original naming the go sdk - I would keep the names as they are to keep the relationship close to the go sdk. Same for moving NewCCPackage to the client.

> The other thought i would have is whether the sgx-mode we might expose also in the request (maybe still with the env-var based derivation if it is not explicitly set in request)?

yes, good idea.

Reply:
> Regarding naming, I was following the original naming the go sdk - I would keep the names as they are to keep the relationship close to the go sdk. Same for moving NewCCPackage to the client.

Oops, forgot about the packager package in standard go and wrongly thought there wouldn't be any packaging in the go framework.  Indeed, your proposal does matches  the various forms from standard sdk.